setup.py

```diff
@@ -105,7 +105,8 @@ def get_requirements():
             'anitya = rebasehelper.versioneers.anitya_versioneer:AnityaVersioneer',
             'pypi = rebasehelper.versioneers.pypi_versioneer:PyPIVersioneer',
             'rubygems = rebasehelper.versioneers.rubygems_versioneer:RubyGemsVersioneer',
-            'npmjs = rebasehelper.versioneers.npmjs_versioneer:NPMJSVersioneer'
+            'npmjs = rebasehelper.versioneers.npmjs_versioneer:NPMJSVersioneer',
+            'cpan = rebasehelper.versioneers.cpan_versioneer:CPANVersioneer'
```

Comment:
Keep the `,` at the end of the last item of a list, it is allowed in python and it is actually [quite convenient](https://stackoverflow.com/questions/11597901/why-does-python-allow-a-trailing-comma-in-list).